src/sentry/api/endpoints/organization_member_invite/index.py

```diff
@@ -0,0 +1,337 @@
+            organization_id=organization.id,
+            target_object=omi.id,
+            data=omi.get_audit_log_data(),
+            event=(audit_log.get_event_id("MEMBER_INVITE")),
+        )
+
+        return Response(serialize(omi), status=201)
+
+    def _request_to_invite_member(self, request: Request, organization) -> Response:
+        serializer = OrganizationMemberInviteRequestSerializer(
+            data=request.data,
+            context={"organization": organization, "allowed_roles": roles.get_all()},
+        )
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+        result = serializer.validated_data
+
+        with transaction.atomic(router.db_for_write(OrganizationMemberInvite)):
+            teams = []
+            for team in result.get("teams", []):
+                teams.append({"id": team.id, "slug": team.slug, "role": None})
+
+            omi = OrganizationMemberInvite(
+                organization=organization,
+                email=result["email"],
+                role=result["orgRole"],
+                inviter_id=request.user.id,
+                organization_member_team_data=teams,
+                invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value,
+            )
+
+            omi.save()
+
+        self.create_audit_entry(
+            request=request,
+            organization_id=organization.id,
+            target_object=omi.id,
+            data=omi.get_audit_log_data(),
+            event=(audit_log.get_event_id("INVITE_REQUEST_ADD")),
+        )
+
+        async_send_notification(InviteRequestNotification, omi, request.user)
+        return Response(serialize(omi), status=201)
+
+    def get(self, request: Request, organization) -> Response:
+        """
+        List all organization member invites.
+        """
+        queryset = OrganizationMemberInvite.objects.filter(organization=organization).order_by(
+            "invite_status", "email"
+        )
+        if not request.access.has_scope("member:write"):
+            queryset = queryset.filter(invite_status=InviteStatus.APPROVED.value)
+
+        return self.paginate(
+            request=request,
+            queryset=queryset,
+            on_results=lambda x: serialize(x, request.user, OrganizationMemberInviteSerializer()),
+            paginator_cls=OffsetPaginator,
+        )
+
+    def post(self, request: Request, organization) -> Response:
+        assigned_org_role = request.data.get("orgRole") or organization_roles.get_default().id
+        billing_bypass = assigned_org_role == "billing" and features.has(
+            "organizations:invite-billing", organization
+        )
+        if not billing_bypass and not features.has(
+            "organizations:invite-members", organization, actor=request.user
+        ):
+            return Response(
+                {"organization": "Your organization is not allowed to invite members"}, status=403
+            )
```

Comment:
it could be cleaner to put this validation in the serializer too, and raise an `APIException` with status_code 403, so we don't have to explicitly check `orgRole` here